datasets/interpress_news_category_tr/interpress_news_category_tr.py

```diff
@@ -0,0 +1,115 @@
+# coding=utf-8
+# Lint as: python3
```

Comment:
Missing license information here:
```
# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
Since this is a dataset from Interpress, we have not yet decided which license should be. Would it be a problem if we update it later?

Reply:
This license covers only the script you've written, not the data 🙂 
If you don't know the license for the data yet, please also set the _LICENSE variable to "unknown", like this:
`_LICENSE = "unknown"`